src/themes/dark.ts

```diff
@@ -14,6 +14,7 @@ export default {
   card: {
     backgroundColor: colors.neutral800,
     textColor: colors.white,
+    border: `0.063em solid ${  colors.darkGray}`,
```

Comment:
`em` in css is a unit that is relative to the font size of it's container.
`rem` in css is a unit that is relative to the front size of the root element.

Neither are really the appropriate units for a border, what do you think of using `1px` here instead?

Reply:
Will do. I thought the use of `em` / `rem` was necessary to make a border's width responsive. However, I just checked and that it isn't the case. I'll keep this in mind.